cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.aws.tsx

```diff
@@ -22,6 +22,9 @@ const initialState = {
         // InstanceType
         NODE_PROFILE: '',
 
+        // Image
+        IMAGE_INFO: {},
```

Comment:
same comment in your other PR about changing name to OS_IMAGE

Reply:
Done

Reply:
Add handle error state by providing a notification "Unable to retrieve a list of OS images: " and include the body of the error text. （The same issue you mentioned in PR #5158